core/miner/storage_fsm/impl/precommit_batcher_impl.hpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <boost/multiprecision/cpp_int.hpp>
+#include <chrono>
+#include <libp2p/protocol/common/asio/asio_scheduler.hpp>
+#include <libp2p/protocol/common/scheduler.hpp>
+#include <map>
+#include <vm/actor/actor.hpp>
```

Comment:
```suggestion
#include "vm/actor/actor.hpp"
```

Reply:
And move it to .cpp as using of MethodParams

Reply:
fixed